packages/dashboard/src/components/TfChainConnector.vue

```diff
@@ -35,8 +35,7 @@
         <v-card-title>Connect your TFChain Wallet</v-card-title>
         <v-card-subtitle>
           Please visit
-          <a href="https://manual.grid.tf/weblets/weblets_profile_manager.html" target="_blank">the manual</a> get
-          started.
+          <a href="https://manual.grid.tf/playground/wallet_connector.html" target="_blank">the manual</a> get started.
```

Comment:
this link should point to the wallet connector on the dashboard section on the docs, not the playground one